docs/supported_code_analyzers.md

```diff
@@ -0,0 +1,30 @@
+## Code Analysis tools
```

Comment:
Maybe we should highlight what we mean by supporting a static analyzer. CodeChecker executes the analyzer or there is a report converter to the analyzer output.